test/quantization/test_qat.py

```diff
@@ -18,8 +18,8 @@
     fake_quantize_per_channel_group,
     fake_quantize_per_token,
 )
-from torchao.quantization.quant_primitives import get_group_qparams_symmetric
-from torchao.quantization.utils import TORCH_VERSION_AFTER_2_4
+from torchao.quantization.utils import get_group_qparams_symmetric
```

Comment:
I'm having trouble figuring out what should be in quant_primitives vs utils

Reply:
functions in utils are mostly helper functions that calls the quant_primitive ops with some fixed parameters, e.g. 

quantize_affine can support: symmetric/asymmetric, per tensor/group/channel etc., int8/int4/int3

helper function in utils can be: int8_symmetric_per_tensor_quant that calls quantize_affine op with fixed settings